Solutions/Palo Alto - XDR (Cortex)/Detection Queries/High Severity Cortex Incidents. yaml

```diff
@@ -0,0 +1,33 @@
+id: be8325a4-530c-484d-b301-e01be23395e2
+name: Palo Alto XDR native Incidents
+description: |
+  'The query checks for all High Log Severity Alerts and incidents fired in Cortex and returns information of enetities.'
+severity: High
+requiredDataConnectors:
+  - connectorId: CommonSecurityLog
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1562
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Palo Alto Networks"
+  | where LogSeverity has_any (2,3,4,5,6)
+  | summarize by TimeGenerated,DeviceProduct, DeviceEventClassID, LogSeverity, Activity, DeviceAction, Computer, DeviceVersion, FileHash, FileName, FilePath, RequestURL, SourceHostName, SourceUserName
+  | sort by TimeGenerated
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: FullName
+        columnName: SourceUserName
+  - entityType: Host
+    fieldMappings:
+      - identifier: FullName
+        columnName: SourceHostName
+version: 1.0.0
```

Comment:
kind: Scheduled

value is missing from the template for all three Analytical Rule.